Mathlib/Topology/Instances/EReal.lean

```diff
@@ -245,4 +245,122 @@ instance : ContinuousNeg EReal := ⟨negOrderIso.continuous⟩
 #align ereal.neg_homeo Homeomorph.neg
 #align ereal.continuous_neg ContinuousNeg.continuous_neg
 
+/-! ### Continuity of multiplication -/
+
+/- Outside of indeterminacies `(0, ±∞)` and `(±∞, 0)`, the multiplication on `EReal` is continuous.
+There are many different cases to consider, so we first prove some special cases and leverage as
+much as possible the symmetries of the multiplication. -/
+
+private lemma continuousAt_mul_swap {a b : EReal}
+    (h : ContinuousAt (fun p : EReal × EReal ↦ p.1 * p.2) (a, b)) :
+    ContinuousAt (fun p : EReal × EReal ↦ p.1 * p.2) (b, a) := by
+  have : (fun p : EReal × EReal ↦ p.1 * p.2) = (fun p : EReal × EReal ↦ p.1 * p.2)
+      ∘ Prod.swap := by
+    ext p
+    simp only [Function.comp_apply, Prod.fst_swap, Prod.snd_swap]
+    exact mul_comm p.1 p.2
+  rw [this]
+  apply ContinuousAt.comp _ (Continuous.continuousAt continuous_swap)
+  simp [h]
```

Comment:
```suggestion
  convert h.comp continuous_swap.continuousAt (x := (b, a))
  simp [mul_comm]
```
Just a little golf. Not really important, but just to show some tricks.

Reply:
Done. Thanks.